Adyen/adyen-magento2
# [PW-8225] Enable PBL for Admin API

**Description**
This PR makes PBL accessible for admin api. This involves updating the can_use_checkout flag for `adyen_pay_by_link` to 1, which results in `adyen_pay_by_link` payment method being returned in the `/payment-methods` call. However, since no entry is added in the `adyen-methods.js` file, PayByLink is still not shown in the frontend.

**Tested scenarios**
- make a /payment-methods call with REST and make sure that PBL is returned in the response
- make sure that on the checkout page of the storefront, PBL is not visible as a payment option
- make sure that you can make a payment using PBL via admin orders
